ss58-registry.json

```
@@ -298,6 +298,15 @@
 			"standardAccount": "*25519",
 			"website": "https://nodle.io/"
 		},
+		{
+			"prefix": 38,
+			"network": "uniarts",
+			"displayName": "UniArts Network",
+			"symbols": ["UART","UINK"],
+			"decimals": [12],
```

```suggestion
			"decimals": [12, 12],
```

I assume both have 12 decimals?